docs/conditions.md

```diff
@@ -34,7 +34,7 @@ The value can be given as a Python `True` or `False` Boolean value or as the str
 
 ### `parameter`
 
-Allows a flag to be enabled based on a GET parameter with the name given as the condition's value.
+Allows a flag to be enabled if a GET parameter with the condition's value as its name exists for a request.
```

Comment:
The wording of this sentence is a little bit unclear at first, and doesn't mention potential subtleties about what the parameter value needs to be. What about:

Allows a flag to be enabled by including a parameter in the request's query string.

```python
FLAGS = {'MY_FLAG': [{'condition': 'parameter', 'value': 'activate'}]}
```

If this flag is checked as part of a web request to a URL like https://example.com/, it will be enabled if the request query string contains the `activate` parameter, regardless of the parameter's value:

https://example.com/?activate
https://example.com/?activate=1
https://example.com/?activate=true
https://example.com/?activate=false

All of the above URLs will enable the `MY_FLAG` flag given use of this condition.